Author: Yves-Alexis Perez <corsac@debian.org>
Author: Lionel Le Folgoc <mrpouit@gmail.com>
Description: adapt GTK+ greeter config to Debian
Especially:

* use background from desktop-base
* force the use of default GTK+ 3 theme (Adwaita)
* use some hinting

Forwarded: not-needed

--- a/data/lightdm-gtk-greeter.conf
+++ b/data/lightdm-gtk-greeter.conf
@@ -16,15 +16,15 @@
 # screensaver-timeout = Timeout (in seconds) until the screen blanks when the greeter is called as lockscreen
 # 
 [greeter]
-#background=
-#theme-name=
+background=/usr/share/images/desktop-base/login-background.svg
+theme-name=Adwaita
 #icon-theme-name=
 #font-name=
-#xft-antialias=
+xft-antialias=true
 #xft-dpi=
-#xft-hintstyle=
-#xft-rgba=
-#show-indicators=
+xft-hintstyle=hintfull
+xft-rgba=rgb
+show-indicators=~language;~session;~power
 #show-clock=
 #clock-format=
 #keyboard=
